### ISSUE TITLE
Issue while running LAMMPS

### ISSUE THREAD

**Issue by mayankaditya** [deepmodeling/deepmd-kit#316]
Hi,

I have trained the AIMD data using deepmd and freeze the model using dp freeze.
While running the LAMMPS, I am getting the following error:

**Invalid argument: NodeDef mentions attr 'T' not in Op<name=DescrptSeA; signature=coord:double, type:int32, natoms:int32, box:double, mesh:int32, davg:double, dstd:double -> descrpt:double, descrpt_deriv:double, rij:double, nlist:int32; attr=rcut_a:float; attr=rcut_r:float; attr=rcut_r_smth:float; attr=sel_a:list(int); attr=sel_r:list(int)>; NodeDef: {{node DescrptSeA}}. (Check whether your GraphDef-interpreting binary is up to date with your GraphDef-generating binary.).
         [[DescrptSeA]]**

I have updated the software using conda, but still getting the same error.
Thanks,
Mayank

**Comment by njzjz** [comment]
I can confirm that the models trained using v1.2.2 cannot be used to run simulations using v1.2.3. I suggest to use the same version.

**Comment by njzjz** [comment]
We renamed v1.2.3 to v1.3.0 to show the correct model compatibility.

**Comment by rajnichahal** [comment]
Hi, I trained a model using version 1.2. and then I converted it to be compatible with the version 2.1. using the following:

dp convert-from 1.2 -i trained_model.pb -o conv_trained_model.pb

However, it is still giving me the following error when running LAMMPS using converted model on version 2.1.:

E tensorflow/core/common_runtime/executor.cc:654] Executor failed to create kernel. Invalid argument: NodeDef mentions attr 'T' not in Op<name=DescrptSeA; signature=coord:double, type:int32, natoms:int32, box:double, mesh:int32, davg:double, dstd:double -> descrpt:double, descrpt_deriv:double, rij:double, nlist:int32; attr=rcut_a:float; attr=rcut_r:float; attr=rcut_r_smth:float; attr=sel_a:list(int); attr=sel_r:list(int)>; NodeDef: {{node DescrptSeA}}. (Check whether your GraphDef-interpreting binary is up to date with your GraphDef-generating binary.).
	 [[DescrptSeA]]
Invalid argument: NodeDef mentions attr 'T' not in Op<name=DescrptSeA; signature=coord:double, type:int32, natoms:int32, box:double, mesh:int32, davg:double, dstd:double -> descrpt:double, descrpt_deriv:double, rij:double, nlist:int32; attr=rcut_a:float; attr=rcut_r:float; attr=rcut_r_smth:float; attr=sel_a:list(int); attr=sel_r:list(int)>; NodeDef: {{node DescrptSeA}}. (Check whether your GraphDef-interpreting binary is up to date with your GraphDef-generating binary.).
	 [[DescrptSeA]]
